packages/legend-extension-dsl-data-product/src/stores/DataProduct/DataProductAccessPointState.ts

```diff
@@ -247,4 +273,67 @@ export class DataProductAccessPointState {
       this.fetchingGrammarState.complete();
     }
   }
+
+  async fetchSampleDataFromEngine(resolvedUserEnv: string): Promise<void> {
+    try {
+      if (this.accessPoint instanceof V1_LakehouseAccessPoint) {
```

Comment:
we should only run this if the access point does not contain sample value provided for by the user. 
also for NOW lets do just access points that are part of an enterprise group. need to get more guidance if we want to do this for every access point regardless of classification